eslint.config.mjs

```diff
@@ -200,6 +198,48 @@ export default tseslint.config([
         },
       ],
 
+      '@stylistic/lines-between-class-members': [
```

Comment:
Nit: alphabetize the rules?

Reply:
Done in [2a271ea](https://github.com/PrairieLearn/PrairieLearn/pull/12371/commits/2a271eaf4f47e83da21d15bb46ae95d7270d242b)